### PR TITLE
Label the specific method in eth_call metrics

### LINKED ISSUE #4979: Perf metrics: Label the specific method of an eth_call
This can help subgraph authors debug which calls are slow.

### PR DESCRIPTION
**Pull request by incrypto32**
Closes #4979

### CODE REVIEW

File: graph/src/components/subgraph/host.rs [hunk 1]
```diff
@@ -108,6 +109,16 @@ impl HostMetrics {
                 vec![0.1, 0.5, 1.0, 10.0, 100.0],
             )
             .expect("failed to create `deployment_handler_execution_time` histogram");
+        let eth_call_execution_time = registry
+            .new_deployment_histogram_vec(
+                "deployment_eth_call_execution_time",
+                "Measures the execution time for eth_call",
+                subgraph,
+                vec![String::from("method"), String::from("contract")],
```
Comment (incrypto32) [inline graph/src/components/subgraph/host.rs:117]: Maybe we should track the contract address also besides the name?

Comment (leoyvens) [reply graph/src/components/subgraph/host.rs:117]: Yes particularly for dynamic data sources I think tracking the address will be helpful

Comment (incrypto32) [reply graph/src/components/subgraph/host.rs]: done

**Review by zorancv — APPROVED** [review]
Looks good to me.

**Pull request merged**